fix escape in ps_1771_update_customer_note()

#### Describe the bug

error on PS update when in note field there is "
“[ERROR] PHP 1.7.7.1 / * PHP: ps_1771_update_customer_note (); * /”

#### Expected behavior

`html_entity_decode($note['note'])`
fix with 
`html_entity_decode(pSQL($note['note']))`

#### Steps to Reproduce

1. install ps 1.7.7.1
2. create user with note that contains quote/double quote
3. update prestashop to latest
4. got error ps_1771_update_customer_note()

**Screenshots**
here's the full code:

```
function ps_1771_update_customer_note()
{
    $notes = Db::getInstance()->executeS(
        'SELECT id_customer, note FROM ' . _DB_PREFIX_ . 'customer
        WHERE note IS NOT NULL AND note != ""'
    );

    $result = true;
    foreach ($notes as $note) {
        $result &= Db::getInstance()->execute(
            'UPDATE ' . _DB_PREFIX_ . 'customer
            SET note = "' . html_entity_decode($note['note']) . '"
            WHERE id_customer = ' . $note['id_customer']
        );
    }

    return (bool) $result;
}
```

It should be:

```
function ps_1771_update_customer_note()
{
    $notes = Db::getInstance()->executeS(
        'SELECT id_customer, note FROM ' . _DB_PREFIX_ . 'customer
        WHERE note IS NOT NULL AND note != ""'
    );

    $result = true;
    foreach ($notes as $note) {
        $result &= Db::getInstance()->execute(
            'UPDATE ' . _DB_PREFIX_ . 'customer
            SET note = "' . html_entity_decode(pSQL($note['note'])) . '"
            WHERE id_customer = ' . $note['id_customer']
        );
    }

    return (bool) $result;
}
```

#### Additional information

* PrestaShop version: 1.7.7.x
* PHP version: 7.2

--- a/install-dev/upgrade/php/ps_1771_update_customer_note.php
+++ b/install-dev/upgrade/php/ps_1771_update_customer_note.php
@@ -38,7 +38,7 @@ function ps_1771_update_customer_note()
     foreach ($notes as $note) {
         $result &= Db::getInstance()->execute(
             'UPDATE ' . _DB_PREFIX_ . 'customer
-            SET note = "' . html_entity_decode($note['note']) . '"
+            SET note = "' . pSQL(html_entity_decode($note['note'])) . '"
             WHERE id_customer = ' . $note['id_customer']
         );
     }

